doc/external-libs/trilinos.html

```diff
@@ -92,6 +93,9 @@ <h5>Installing Trilinos</h5>
     -DTrilinos_ENABLE_MueLu=ON                       \
     -DTrilinos_ENABLE_ML=ON                          \
     -DTrilinos_ENABLE_ROL=ON                         \
+    -DTrilinos_ENABLE_Tpetra=ON                      \
+    -DTpetra_INST_COMPLEX_DOUBLE=ON                  \
+    -DTpetra_INST_COMPLEX_FLOAT=ON                   \
```

Comment:
don't we need `Tpetra_INST_INT_UNSIGNED` and `Tpetra_INST_INT_UNSIGNED_LONG`? I recall by default TPetra does not instantiate for unsigned ints (be it 32 or 64 bits).

Reply:
All I can say right now is that I don't have these activated and its working.